tests/unit/test_hyper_transformer.py

```diff
@@ -1,101 +1,676 @@
 from unittest import TestCase
-from unittest.mock import Mock
+from unittest.mock import Mock, call, patch
 
-import numpy as np
 import pandas as pd
 import pytest
 
 from rdt import HyperTransformer
 from rdt.transformers import (
-    BooleanTransformer, DatetimeTransformer, NumericalTransformer, OneHotEncodingTransformer)
+    BooleanTransformer, CategoricalTransformer, DatetimeTransformer, GaussianCopulaTransformer,
+    NumericalTransformer, OneHotEncodingTransformer)
 
 
 class TestHyperTransformer(TestCase):
 
-    def test___init__(self):
+    def test__validate_field_transformers(self):
+        """Test the ``_validate_field_transformers`` method.
+
+        Tests that a ``ValueError`` is raised if a field has multiple
+        definitions in the ``field_transformers`` dict.
+
+        Setup:
+            - field_transformers set to have duplicate definitions.
+
+        Expected behavior:
+            - A ``ValueError`` should be raised if a field is defined
+            more than once.
+        """
+        # Setup
+        int_transformer = Mock()
+        float_transformer = Mock()
+
+        field_transformers = {
+            'integer': int_transformer,
+            'float': float_transformer,
+            ('integer',): int_transformer
+        }
+        ht = HyperTransformer()
+        ht.field_transformers = field_transformers
+
+        # Run / Assert
+        with pytest.raises(ValueError):
+            ht._validate_field_transformers()
+
+    @patch('rdt.hyper_transformer.HyperTransformer._create_multi_column_fields')
+    @patch('rdt.hyper_transformer.HyperTransformer._validate_field_transformers')
+    def test___init__(self, validation_mock, multi_column_mock):
         """Test create new instance of HyperTransformer"""
         # Run
         ht = HyperTransformer()
 
         # Asserts
         self.assertTrue(ht.copy)
```

Comment:
nit: Since we are already modifying this, I would change the `self.assertXXX` (unit test style) to `assert XXX` (pytest style)